lib/filter.js

```diff
@@ -11,12 +11,45 @@ exports.parseFilter = (filter) => {
     throw new Error('Invalid filter operator');
   }
 
+  if (filter.hasOwnProperty('matchMissing') && !(typeof filter.matchMissing === 'boolean')) {
```

Comment:
There you go: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Boolean :)